projects/core/src/auth/user-auth/events/login-event.builder.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { StateEventService } from '../../../state/event/state-event.service';
+import { AuthActions } from '../store/actions/index';
+import { LoginEvent } from './user-auth.events';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoginEventBuilder {
+  constructor(protected stateEventService: StateEventService) {
+    this.register();
+  }
+
+  /**
+   * Registers logout events
+   */
+  protected register(): void {
+    this.loginEvent();
+  }
+
+  /**
+   * Register a logout event
+   */
+  protected loginEvent(): void {
```

Comment:
To me this is a confusing method name. I'd rather make it `registerEvent()`, `buildEvent()` or `buildLoginEvent()`.

Reply:
And why not add the logic in the `register` method? It's small and to the point.